assets-src/js/translations-sorting.js

```diff
@@ -0,0 +1,36 @@
+const userLangs = navigator.languages || [navigator.language || navigator.userLanguage];
```

Comment:
add description

Reply:
addressed in https://github.com/w3c/w3c-website-templates-bundle/pull/96/commits/0c618d962d9f428f3bd13166c2d62b4bb549bf06